packages/utils/src/content.ts

```diff
@@ -0,0 +1,17 @@
+import { CodegenSchema, isCodegenObjectLikeSchema, isCodegenArraySchema } from '@openapi-generator-plus/types'
+
+/**
+ * Return the "value" schema for a metadata schema that might be an object schema or array schema.
+ * Otherwise return null if such a schema cannot be found.
+ * @param schema 
+ * @returns 
+ */
+export function valueSchemaForMetadataSchema(schema: CodegenSchema): CodegenSchema | null {
```

Comment:
Actually, I think in the core generator we'd usually return `CodegenSchema | undefined` from functions. The `null` thing is something for the object properties themselves.